app/controllers/api/account.php

```diff
@@ -494,6 +498,53 @@
                         'memberships' => null,
                         'search' => implode(' ', [$userId, $email, $name])
                     ])));
+
+                    /** Trigger Webhook */
+                    $userModel = new User();
+
+                    $userCreate = new Event(Event::WEBHOOK_QUEUE_NAME, Event::WEBHOOK_CLASS_NAME);
+                    $userCreate
+                        ->setProject($project)
+                        ->setUser($user)
+                        ->setEvent('users.[userId].create')
+                        ->setParam('userId', $userId)
+                        ->setPayload($user->getArrayCopy(array_keys(($userModel->getRules()))))
```

Comment:
What's the reasoning on this rather than just using `$user`?

Reply:
To prevent data such as the password hash from being accidentally exposed